articles/custom-domains/feature-specific-setup.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Feature-Specific Setup with Custom Domains
+description: How to set up custom domains with feature-specific instructions
+toc:true
+---
+# Feature-Specific Setup
+
```

Comment:
Is it possible to add a TOC for this page? It might make it easier to navigate. Wdyt?

Reply:
Yeah, that's what the commit just before this comment does :)

Gives the reader the "In this Article" TOC

Reply:
Yes, definitely (at first I was wondering what you meant, since one was added, but we're on a different page, heh)